### DISCUSSION TITLE
Mail authentication

### DISCUSSION THREAD

**Discussion by Ijaiswalshivam** [Ijaiswalshivam/Talk-In#101] [Q&A]
I changed the json file and done the dependency setup, still I'm not able to get mail for verification, always showing "The email address is already in use", kindly help me to rectify this problem

**Comment by Ijaiswalshivam** [comment]
@NitinSigilipelli can you please help me?

**Comment by NitinSigilipelli** [comment]
Check once which project json you dealing with and delete if you have any of your mail in authentication users in firebase then try to signup through our app

**Reply by Ijaiswalshivam** [reply]
here all are correct

**Comment by NitinSigilipelli** [comment]
Did you get your verification mail or not ?

**Reply by Ijaiswalshivam** [reply]
not

**Comment by NitinSigilipelli** [comment]
After deleting all users in authentication still getting same problem ?

**Reply by Ijaiswalshivam** [reply]
yeah, after all adding new mail, still same response

**Reply by Ijaiswalshivam** [reply]
you can clone and check it again

**Comment by NitinSigilipelli** [comment]
Yeah ok i will check by EOD
